test/e2e/reloadinterval_test.go

```diff
@@ -73,9 +73,9 @@ func TestReloadInterval(t *testing.T) {
 
 		// set ReloadInterval to a value outside allowed bounds
 		// expect no error
-		{"set reloadInterval 200ns", metav1.Duration{Duration: 200 * time.Nanosecond}, "5s", true},
-		{"set reloadInterval 5us", metav1.Duration{Duration: 5 * time.Microsecond}, "5s", true},
-		{"set reloadInterval 500ms", metav1.Duration{Duration: 500 * time.Millisecond}, "5s", true},
+		{"set reloadInterval 200ns", metav1.Duration{Duration: 200 * time.Nanosecond}, "1s", true},
+		{"set reloadInterval 5us", metav1.Duration{Duration: 5 * time.Microsecond}, "1s", true},
```

Comment:
does setting to 1s  really help ? Won't  it cause load on the router ?

Reply:
Not exactly - it's the value the test is looking for in the router deployment ENV variable. So no load affects, just a comparison value that was wrong.